example-jobs/SampleBuildFlow-example.groovy

```diff
@@ -44,7 +44,9 @@ customFlowJobWithWorkspace.with{
         git('https://github.com/cfpb/qu.git', 'master')
     }
 
+    //for use on job-dsl-plugin versions 1.41 and earlier
+    //on 1.42+, simply use buildNeedsWorkspace(). See https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.jobs.BuildFlowJob.buildNeedsWorkspace
```

Comment:
Could you modify the formatting of this comment to clarify whether or not the period at the end of `buildNeedsWorkspace().` is necessary, or just separating clauses in the comment?